Runbooks/startstop/1.1/Bootstrap_Main.ps1

```diff
@@ -0,0 +1,315 @@
+﻿<#
+.SYNOPSIS
+ Bootstrap master script for pre-configuring Automation Account
+.DESCRIPTION
+ Bootstrap master script for pre-configuring Automation Account
+.EXAMPLE
+.\Bootstrap_Main.ps1
+Version History
+v1.0 - Initial Release
+v2.0 - Refactored Az modules
+#>
+
+
+# ------------------Execution Entry point ---------------------
+
+Write-Output 'Bootstrap main script execution started...'
+
+#---------Inputs variables for NewRunAsAccountCertKeyVault.ps1 child bootstrap script--------------
+$automationAccountName = Get-AutomationVariable -Name 'Internal_AutomationAccountName'
+$SubscriptionId = Get-AutomationVariable -Name 'Internal_AzureSubscriptionId'
```

Comment:
Never used

Reply:
Not important. I will leave it as is for now.